package.json

```diff
@@ -7,15 +7,15 @@
     "start": "next start",
     "deploy": "yarn && next build && yarn --production=true && sls deploy --aws-profile hnserverless -s staging && yarn",
     "sls-debug": "export SLS_DEBUG=* && node --inspect ./node_modules/serverless/bin/serverless offline -s staging --noAuth --noPrependStageInUrl",
-    "cypress-open": "npx cypress open",
+    "cypress-open": "npx cypress open --env AWS_SECRET_ACCESS_KEY=bar,AWS_REGION=localhost,AWS_ACCESS_KEY_ID=foo",
     "cypress-run": "cypress run",
-    "int-test": "npm run build && start-server-and-test start http://localhost:3000 cypress-run",
+    "preint-test": "yarn build",
```

Comment:
did this not work with the chained command?

Reply:
It works with the chained command. We changed two bits here:
1. The chained command was using `npm run build`, but from Circle we were triggering this with `yarn run int-test`. Technically `yarn` and `npm` can use different versions of node which may be problematic and was triggering warnings, and so we switched to `yarn build` instead for consistency.
2. We switched from a chained command to a `preint-test` script. This will run automatically before `int-test` and if it becomes more complicated it's not getting in the way of trying to understand what `int-test` itself is doing.